src/style.css

```
@@ -447,6 +447,361 @@ body {
+}
+
+.skill-item.enabled .skill-name {
+  color: #4ade80;
+}
+
+.skill-preview {
+  color: #6b7280;
+  font-size: 11px;
+  line-height: 1.5;
+  white-space: pre-wrap;
+  word-break: break-word;
```

Stylelint (17.6.0)

[error] Unexpected deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)